WordPress/Jetpack/Classes/ViewRelated/WordPress-to-Jetpack Migration/Common/Views/Configuration/MigrationHeaderConfiguration.swift

```diff
@@ -96,7 +96,7 @@ private extension MigrationHeaderConfiguration {
                                                               comment: "Primary description in the migration done screen.")
 
         static let notificationsSecondaryDescription = NSLocalizedString("migration.notifications.secondaryDescription",
-                                                                         value: "We’ve disabled notifications for the WordPress app.",
+                                                                         value: "We’ll disable notifications for the WordPress app.",
```

Comment:
> I noticed that the body of the push notification should be fetched from the server when the notification arrives ([see code](https://github.com/wordpress-mobile/WordPress-iOS/blob/784c2a4f2ad74deb36da0a20d84b6fdd16ed56b6/WordPress/WordPressNotificationServiceExtension/Sources/NotificationService.swift#L118)). After going through the migration flow, this code isn't reached because it aborts when calling readExtensionToken() ([see code](https://github.com/wordpress-mobile/WordPress-iOS/blob/784c2a4f2ad74deb36da0a20d84b6fdd16ed56b6/WordPress/WordPressNotificationServiceExtension/Sources/NotificationService.swift#L39)).

What an important catch that is! 👏 Thanks for the attention to detail.

Reply:
>  I'm not sure if it's because the timing of translations has changed or I just never ran into this.

I'll check it out.